aptamers.py

```diff
@@ -243,7 +243,10 @@ def get_parents(self, state=None, done=None):
             parents = []
             actions = []
             for idx, a in enumerate(self.action_space):
-                if state[-len(a) :] == list(a):
+                is_parent = state[-len(a) :] == list(a)
+                if not isinstance(is_parent, bool):
+                    is_parent = all(is_parent)
```

Comment:
This is to adapt to the case when the action can be of len > 1, right ? :)

Reply:
Exactly!